cross/python38/PLIST

```diff
@@ -1,14 +1,15 @@
 lnk:bin/2to3
-rsc:bin/2to3-3.8
+rsc:bin/2to3-@PKG_SHORT_VERS@
 lnk:bin/idle3
-rsc:bin/idle3.8
+rsc:bin/idle@PKG_SHORT_VERS@
 lnk:bin/pydoc3
-rsc:bin/pydoc3.8
+rsc:bin/pydoc@PKG_SHORT_VERS@
 lnk:bin/python3
-bin:bin/python3.8
+bin:bin/python@PKG_SHORT_VERS@
 rsc:etc/mime.types
-rsc:include/python3.8/pyconfig.h
-lnk:lib/libpython3.8.so
-lib:lib/libpython3.8.so.1.0
+rsc:include/python@PKG_SHORT_VERS@/pyconfig.h
+lnk:lib/libpython@PKG_SHORT_VERS@.so
+lib:lib/libpython@PKG_SHORT_VERS@.so.1.0
 lib:lib/libpython3.so
-rsc:lib/python3.8
+rsc:lib/python@PKG_SHORT_VERS@
```

Comment:
This will not work without a custom PLIST_TRANSFORM definition.

I intentionally changed this from @PKG_SHORT_VERS@ to 3.8 when creating the python38 package as we never will update python 3.8 to 3.9. Instead we will create a dedicated python39 package.